internal/rpc/clients.go

```diff
@@ -310,6 +312,11 @@ func newGRPCDialOptions(enableMetrics bool, enableRPCLogging bool, enableRPCPayl
 		grpc.WithStreamInterceptor(grpc_middleware.ChainStreamClient(si...)),
 		grpc.WithUnaryInterceptor(grpc_middleware.ChainUnaryClient(ui...)),
 		grpc.WithBalancerName(roundrobin.Name),
+		grpc.WithKeepaliveParams(keepalive.ClientParameters{
```

Comment:
PR description states "Moved the KeepAliveParams from client side to server side"

This does the opposite.

Reply:
oops that's a typo